scripts/exportTailwindColors.cjs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
Just curious: What is a `.cjs` file?
This seems to be a JS file and how does it have to be run? Ccould that be documented?

In my personal favour, an import that is Julia-based would even be cooler, if possible. Since this seems to not to a HTTP call or so, maybe the loading of coolers has to be done in JS though?

Reply:
> What is a `.cjs` file?

`.cjs` files are functionally JavaScript files, _but_ convey that imports are done via `CommonJS` (which is what `node` understands by default).

> ... how does it have to be run? Could that be documented?

👀 I thought I did this. 😅 (I see that I in-fact did not.) It could definitely be documented. Though let's circle-back after this detour down using Julia x NodeJS.

I’d also love to use a Julia script to do this. 😅

> Since this seems to not to a HTTP call or so, maybe the loading of coolers has to be done in JS though?

So, the file that contains this color data is [available here][tailwind-colors]. Though it's mostly JSON-like, there are a few bits that would break every JSON parser I'm familiar with. (I'd also prefer to avoid doing some RegEx-like matching since `colors` has reliably exported by Tailwind for the last two major versions.)

However, I see now that [`NodeJS.jl`][nodejs.jl] exists, so it's possible I could reimplement the update script in Julia by-proxy? Unless, you were thinking of a pure Julia implementation (I'm not sure how feasible this would be, sadly.)

[nodejs.jl]: https://github.com/davidanthoff/NodeJS.jl
[tailwind-colors]: https://github.com/tailwindlabs/tailwindcss/blob/master/src/public/colors.js

Reply:
With “A file in julia” I was thinking of a Julia script that (as independent as possible) updates the colors with something like `julia update-tailwind.jl` – not that you should implement that completely yourself, but with `NodeJS.jl` for example.

Reply:
Ah. I didn't give `NodeJS.jl` a proper look. So, all it appears to do is provide a wrapper for Julia to run `node` and `npm` commands, so the JS code seems it would still need to exist. I think the alternative would be to roll a brittle solution that is liable to break if the folks at Tailwind ever change the path/format of `colors`.

Reply:
I would also be fine with a thorough `Readme` how to update the colors, so this can be done, without reaching out to you ;) I thought a Julia script might be the easiest.

Reply:
I am open for any solution, as long as a non-JS person as me has an easy way to run the update :)

Reply:
Something I just considered (since Tailwind recently updated the colors). The colors are subject to change – I'm not sure if other colors in `NamedColors` are liable to change (often). (I've been using Tailwind for the last 4 years and this is the first time I recall it changing, so it might not be a very frequent thing.)

So, this leads me to: NodeJS has a way to install specific versions (via the `package.json`) or we can install the latest Tailwind colors. **Is there a preferred route for `NamedColors`?**

Personally, I'm okay with the potentially moving target (installing the latest release of `tailwindcss`), but I dunno if there's some existing thoughts in the context of `NamedColors` about this.

Reply:
Since it would be a manual update triggered by a user, I think it is reasonable to take the latest release. If that changes, the user that has a reason to manually update them should also do a PR with a new version (a breaking one).